modules/media/php/media.class.inc

```diff
@@ -141,8 +149,10 @@ class Media extends \NDB_Menu_Filter
     {
         $user =& \User::singleton();
         // the base query
-        $query  = " FROM media m LEFT JOIN session s ON m.session_id = s.ID".
-        " LEFT JOIN candidate c ON c.CandID=s.CandID";
+        $query  = " FROM media m 
+                    LEFT JOIN session s ON m.session_id = s.ID 
+                    LEFT JOIN candidate c ON c.CandID=s.CandID 
+                    LEFT JOIN language l USING (language_id) ";
         $query .= " WHERE 1=1 ";
```

Comment:
Why is this line necessary?

Reply:
I haven't looked into this module in particular, but many modules include a "WHERE 1=1" clause at the end so that when further clauses get added with "... AND x=foo", it doesn't result in a syntax error.